docker-compose.yml

```diff
@@ -59,6 +61,7 @@ services:
     networks:
       - gb-backend-db-network
       - nginx-proxy-network
+    restart: unless-stopped
 
   # ----- transmart-api-server + db -----
   transmart-database:
```

Comment:
the restart policy change is skipped for the `transmart-database` service. It would be useful to add it there as well.